src/coreclr/pal/src/exception/signal.cpp

```diff
@@ -653,7 +653,6 @@ static void sigsegv_handler(int code, siginfo_t *siginfo, void *context)
             else
```

Comment:
Nit: The "The current executable (shared library) doesn't have...." comment can be moved after the `if ... else `. It applies to the else part too now.

Reply:
Makes sense, moved.